policyengine_us/parameters/gov/states/nc/tax/use_tax/amount.yaml

```diff
@@ -0,0 +1,138 @@
+description:  North Carolina Use Tax using this amount for taxpayers without out-of-state purchases records.
+metadata:
+  type: single_amount
+  threshold_unit: currency-USD
+  amount_unit: currency-USD
+  label: NC Use Tax amount cap for different taxable income
+  reference:
+    - title: NCDOR Use Tax
+      href: https://www.ncdor.gov/taxes-forms/sales-and-use-tax/use-tax
+    - title: 2022 NC D-400
+      href: https://www.ncdor.gov/2022-d-400-web-fill-version/open
+    - title: 2022 D-401 Instruction Booklet
+      href: https://www.ncdor.gov/2022-d-401-individual-income-tax-instructions/open
```

Comment:
add page number for easy reference on document
i.e https://www.ncdor.gov/2022-d-401-individual-income-tax-instructions/open#page=23
This link will take you to excactly page 23 of the document